iocage/Resource/__init__.py

```diff
@@ -413,3 +413,16 @@ def destroy(
     def save(self) -> None:
         """Save changes to the default configuration."""
         self._write_config(self.config.user_data)
+
+
+iocage.hook_module(
```

Comment:
is this a python thing, or a (lib)iocage thing?

Reply:
A local workaround for ugly Python module names. Makes them callable and act like classes with the same names.